File: Modul 4/Modul 4 Bahasa python/PRAK405-2410817310007-MuhammadNaufalAbdillah.py
baris1,baris2=map(int,input().split())
total=0
for i in range(1,baris1 +1):
    hasilbaris=0

    for j in range(i,0, -1):
        hasilbaris+=j*baris2
        print(f"({j}*{baris2})",end="")
        if j>1 :
            print("+",end="")

    print(f"={hasilbaris}")
    total+=hasilbaris

print(total)

File: Modul 4/Modul 4 Bahasa C/PRAK405-2410817310007-MuhammadNaufalAbdillah.c
#include <stdio.h>
int main()
{
    int baris1,baris2;
int total=0;
scanf("%d %d",&baris1, &baris2);
for(int i=1; i<=baris1; i++){
    int hasilbaris=0;

    for(int j=i;j>=1; j--)
    {
        hasilbaris+=j*baris2;
        printf("(%d*%d)",j,baris2);
        if(j>1)
        {
         printf("+");
        }
    }
printf("= %d\n",hasilbaris);
total+=hasilbaris;}
printf("%d",total);

    return 0; 
}